test/e2e/coverage-metrics/create-test-session.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
these new scripts are currently not used in the workflow right? is the workflow going to be updated in another PR?

Reply:
This script will be used in a workflow that will run tests periodically. Once we decide are stable and agree to run them in prs then we can add them to prs checks

Reply:
could you link the workflow run where these scripts were used? (I guess in your fork of this repo?)

Reply:
Ungortunetly I deleted the workflows in my main fork